first-party/jib-layer-filter-extension-maven/src/test/java/com/google/cloud/tools/jib/maven/extension/layerfilter/JibLayerFilterExtensionTest.java

```diff
@@ -291,4 +326,115 @@ public void testExtendContainerBuildPlan_complex() throws JibPluginExtensionExce
         Arrays.asList("/alpha/Bob", "/beta/Bob", "/gamma/Bob"), layerToExtractionPaths(newLayer5));
     assertEquals(Arrays.asList("/gamma/Charlie"), layerToExtractionPaths(newLayer6));
   }
+
+  @Test
+  public void testExtendContainerBuildPlan_createParentLayers_noProjectDependenciesResolver() {
+    ContainerBuildPlan buildPlan = ContainerBuildPlan.builder().build();
+    Configuration cfg = new Configuration();
```

Comment:
```suggestion
    Configuration config = new Configuration();
```
and in other tests too.

Reply:
ok